infra/feast-operator/test/e2e/e2e_test.go

```diff
@@ -36,70 +37,79 @@ const (
-		// projectimage stores the name of the image used in the example
-		var projectimage = "localhost/feast-operator:v0.0.1"
-
-		By("building the manager(Operator) image")
-		cmd = exec.Command("make", "docker-build", fmt.Sprintf("IMG=%s", projectimage))
-		_, err = utils.Run(cmd)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("loading the the manager(Operator) image on Kind")
-		err = utils.LoadImageToKindClusterWithName(projectimage)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("building the feast image")
-		cmd = exec.Command("make", "feast-ci-dev-docker-img")
-		_, err = utils.Run(cmd)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-		// this image will be built in above make target.
-		var feastImage = "feastdev/feature-server:dev"
-		var feastLocalImage = "localhost/feastdev/feature-server:dev"
-
-		By("Tag the local feast image for the integration tests")
-		cmd = exec.Command("docker", "image", "tag", feastImage, feastLocalImage)
-		_, err = utils.Run(cmd)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("loading the the feast image on Kind cluster")
-		err = utils.LoadImageToKindClusterWithName(feastLocalImage)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("installing CRDs")
-		cmd = exec.Command("make", "install")
-		_, err = utils.Run(cmd)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("deploying the controller-manager")
-		cmd = exec.Command("make", "deploy", fmt.Sprintf("IMG=%s", projectimage), fmt.Sprintf("FS_IMG=%s", feastLocalImage))
-		_, err = utils.Run(cmd)
-		ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-		By("Validating that the controller-manager deployment is in available state")
-		err = checkIfDeploymentExistsAndAvailable(feastControllerNamespace, controllerDeploymentName, timeout)
-		Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
-			"Deployment %s is not available but expected to be available. \nError: %v\n",
-			controllerDeploymentName, err,
-		))
-		fmt.Printf("Feast Control Manager Deployment %s is available\n", controllerDeploymentName)
+		isRunOnOpenShiftCI := os.Getenv("RUN_ON_OPENSHIFT_CI")
+		if isRunOnOpenShiftCI == "" {
+			By("creating manager namespace")
+			cmd := exec.Command("kubectl", "create", "ns", feastControllerNamespace)
+			_, _ = utils.Run(cmd)
+
+			var err error
+			// projectimage stores the name of the image used in the example
+			var projectimage = "localhost/feast-operator:v0.0.1"
+
+			By("building the manager(Operator) image")
+			cmd = exec.Command("make", "docker-build", fmt.Sprintf("IMG=%s", projectimage))
+			_, err = utils.Run(cmd)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+			By("loading the the manager(Operator) image on Kind")
+			err = utils.LoadImageToKindClusterWithName(projectimage)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+			// this image will be built in above make target.
+			var feastImage = "feastdev/feature-server:dev"
+			var feastLocalImage = "localhost/feastdev/feature-server:dev"
```

Comment:
I wonder how opnshift-ci gets the feast image if we skip this section also.

Reply:
i believe openshift CI uses its own builds & image repo